### ISSUE TITLE
Simplify doesn't work with log or ln

### ISSUE THREAD

**Issue by pjhampton** [josdejong/mathjs#849]
**Example Query**

`math.eval(math.simplify('log(10,10)'))`

**Stack Trace**

```
math.min.js:31 Uncaught Error: Invalid symbol in rule: e
    at p (math.min.js:31)
    at p (math.min.js:30)
    at Node, Object (math.min.js:31)
    at applyRule (eval at h (math.min.js:53), <anonymous>:12:16)
    at Node, Array (math.min.js:31)
    at simplify (eval at h (math.min.js:53), <anonymous>:49:16)
    at string (math.min.js:31)
    at Object.simplify (eval at h (math.min.js:53), <anonymous>:23:14)
    at <anonymous>:1:16
```

**Comment by josdejong** [comment]
Thanks for reporting, this is a duplicate of #838 and has just been fixed via #840. Will be available in the first next release.

**Comment by pjhampton** [comment]
Ahh Thanks @josdejong 👍
